### ISSUE TITLE
anti-TPO antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7351]
**Preferred term label:**
anti-TPO antibody positivity

**Synonyms**
anti-Thyroid peroxidase antibody positivity, TPOAbs

**Definition (free text, please give PubMed ID)**
PMID:33796145, 33144894

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Graves' thyroiditis, Hashimoto's thyroiditis

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by MaudDLVT** [comment]
@julienN75 for follow up

**Comment by pnrobinson** [comment]
This term already exists as [Anti-thyroid peroxidase antibody positivity HP:0025379](https://hpo.jax.org/app/browse/term/HP:0025379).
I will add the synoynm and the PMIDs.
Thanks!
